website/source/docs/connect/discovery-chain.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "docs"
+page_title: "Connect - Discovery Chain (beta)"
+sidebar_current: "docs-connect-discovery_chain"
+description: |-
+  A discovery chain is a series of distinct stages that a service discovery request flows through to discover and configure Connect proxy upstreams of the service type.
+---
+
+# Discovery Chain <sup>(beta)</sup>
+
+-> **Note:** This feature is not compatible with the 
+[built-in proxy](/docs/connect/proxies/built-in.html)
+or [native proxies](/docs/connect/native.html).
+
+A discovery chain is a series of distinct stages that a service discovery
+request flows through to discover and configure Connect proxy upstreams of the
+`service` type.
+
+![diagram of discovery chain stages](/assets/images/discovery-chain-simple.svg)
+
+Each stage of this discovery chain can be dynamically reconfigured via various
+[configuration entries](/docs/agent/config_entries.html). When a configuration
+entry is missing, that stage will fall back on reasonable default behavior.
+
+## Terminology
+
+### Service Subsets
+
+There are many ways you may wish to carve up a single datacenter's pool of
```

Comment:
This is great. I'd lead with this paragraph as the first one on the page, and explain everything else with the context of enabling these use cases :D